src/connectors/aws_inventory.py

```diff
@@ -144,7 +146,20 @@
         ('load_balancer_name', 'STRING(256)'),
         ('region_name', 'STRING(16)'),
         ('scheme', 'STRING(30)'),
-        ('vpc_id', 'STRING(30)')
+        ('vpc_id', 'STRING(30)'),
+        ('account_id', 'STRING(30)'),
+    ],
+    # IAM Users
+    'IAM': [
+        ('raw', 'VARCHAR'),
+        ('ingested_at', 'TIMESTAMP_LTZ'),
+        ('path', 'VARCHAR'),
+        ('user_name', 'VARCHAR'),
+        ('user_id', 'VARCHAR'),
+        ('arn', 'VARCHAR'),
+        ('create_date', 'TIMESTAMP_LTZ'),
+        ('password_last_used', 'TIMESTAMP_LTZ'),
+        ('account_id', 'STRING(32)'),
```

Comment:
Why is the length for IAM account_id 32, when it's 30 in the other tables?

Reply:
not sure. @edulop91 should we update the others, as well?

Reply:
Power of 2 seemed nice, not sure if it has any real implications though. Should I make all of them 30?

Reply:
Last we discussed Greg likes powers of 2 and I think they're overrated, but don't think any of us have a strong preference :-)

Reply:
I'm mostly of the opinion that consistency is better than truth - I don't have strong opinions on how long the column should be once we get past basic issues like "is it long enough" but i firmly believe that if two columns are meant to hold the same thing they should be the same size, whatever that size is.